TIER: configure 0:08.91   File "thunderbird-151.0/comm/build/moz.configure/gecko_source.configure", line 54, in get_sourcestamp
TIER: configure 0:08.91     log.warn("sourcestamp.txt is corrupt!")
TIER: configure 0:08.91     ^^^^^^^^
TIER: configure 0:08.91 AttributeError: 'ReadOnlyNamespace' object has no attribute 'warn'
TIER: configureTIER: configure 0:09.08 E *** Fix above errors and then restart with "./mach build"

--- thunderbird-151.0/comm/build/moz.configure/gecko_source.configure.old
+++ thunderbird-151.0/comm/build/moz.configure/gecko_source.configure
@@ -51,7 +51,7 @@
                 pass
 
             if len(lines) != 3:
-                log.warn("sourcestamp.txt is corrupt!")
+                log.info("sourcestamp.txt is corrupt!")
                 return
 
             if lines and lines[line2read].startswith("http"):
